app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> [None, str]:
+    command_split = command.split(" ")
+    action = command_split[0]
+    source_file_option = command_split[1]
+    target_file_option = command_split[2]
+    target_type = None
+    target_folders_path = None
+
+    if action != "mv":
+        print(f"Wrong action is used{action}. mv action is expected")
+
+    if "/" in target_file_option:
+        target_folders_path = target_file_option.rsplit("/", 1)[0]
+        target_type = "folder"
+
+    if "/" not in target_file_option:
+        target_folders_path = target_file_option
+        target_type = "file"
```

Comment:
you can use 
`os.path.isdir(path)` or `os.path.isfile(path)` to know what it is